code/perception/src/EKFTranslationNode.py

```diff
@@ -117,7 +144,10 @@ def update_gps_data(self, data: NavSatFix):
         self.avg_gps_counter = 1
 
         x, y, z = self.transformer.gnss_to_xyz(avg_lat, avg_lon, avg_alt)
-
+        # -> temporary fix todo: find reason for inaccuracy
```

Comment:
This looks like it will be a problem if we drive too far from the starting point. Could it be solved with a fixed offset?

Reply:
Yes, this could definitely be a problem later. However, I tested this temporary fix for points within a 300m radius of the current position and the factor applied was still correct. Nevertheless, this should only be a temporary solution.